free-1.md

```diff
@@ -4,47 +4,143 @@
+need or want all the APIs the javascript language itself provides. Their
+experience right now is writing and reading very long files that really are not
+that complex. But because of all the "FormComponent" and "createClass" funcs,
+it takes long to write. Worst of all, it is hard to come back to and debug
+because of the unnecessary complexity.
+
+These people's experience with web development would be nicer, easier, and they
+will be faster. If they don't need to close every tag, if they can just write
+shorter functions and keywords that can be translated into normal (and longer)
+HTML, CSS, and javascript, then their lives would be easier and more efficient.
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+Because the DSL would be adding an abstraction layer over what they do now. The
+DSL would allow the users to not have to touch the bare-metal code, but instead
+write things that are shorter (i.e. shortcuts) that can be translated/compiled
+into the correct code. It addressed the need by shortening the development time
+and making code in this format nicer to read and debug.
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
 
+This project idea excites me because I've written a lot of HTML, CSS, and JS in
+the past year. I've explored many tools and framework that make things easier.
+A good example is React. With React, I used the JSX syntax, which I thought was
+a good abstraction that made code shorter to write than if I had written the
+same things in plain javascript. However, it also didn't lose any meaning - once
+I got the translation/abstraction, the code was cleaner to read, shorter, and
+easier to debug. However, JSX is mostly accessible in React. Also, it would be
+a little different (in my mind) if something similar could be used to write
+html, css, and javascript without the React framework (as JSX is very custom for
+React and things related to how React is used), so more work would be needed and
+it would not just be a simple port from JSX.
 
 ### Domain
 _Describe the project's domain in five words._
 
+Making webdev easier and nicer.
 
 ### Interface (syntax)
-_How might the user interact with the language? What does programming look 
-like? Why is this the right way to interact with the problem domain?_ 
-
+_How might the user interact with the language? What does programming look
+like? Why is this the right way to interact with the problem domain?_
+
+The user will interact with the language through coding. Programming would look
+like the user is writing a dialect/hybrid of HTML, CSS, and JavaScript, but with
+nicer syntax. This is the right way to interact with the problem domain because
+people will still need to write code, but a DSL can make that simpler and nicer.
+Since they are already writing code, they are not the people that use Wix or
+Squarespace to create websites through drag-and-drop, but they would not be
```

Comment:
I imagine there's some kind of tradeoff space here between usability (drag-and-drop) and power (coding). What kind of balance are you attempting to strike with your DSL? Do you see people familiar with drag-and-drop interfaces maybe learning to code because your language is that much easier to use than existing solutions? Or is it more just about making it easier for existing coders?